qcodes/utils/helpers.py

```diff
@@ -581,3 +582,32 @@ def attribute_set_to(object_: Any, attribute_name: str, new_value: Any):
     setattr(object_, attribute_name, new_value)
     yield
     setattr(object_, attribute_name, old_value)
+
+
+def sane_partial(func, docstring, **kwargs):
```

Comment:
Could you give this a more descriptive name. Sane_partial tells very little about why this is more sane